src/SmartComponents/Advisories/Advisories.test.js

```diff
@@ -55,152 +59,66 @@ const mockState = { ...storeListDefaults,
-
-        const dispatchedActions = store.getActions();
-        expect(dispatchedActions.filter(item => item.type === 'EXPAND_ADVISORY_ROW')).toHaveLength(1);
-    });
-
-    describe('test exports',  ()  => {
-
-        global.Headers = jest.fn();
-        global.fetch = jest.fn(() => Promise.resolve({ success: true }).catch((err) => console.log(err)));
-
-        it('Should download csv file', () => {
-            wrapper.find('TableView').props().onExport(null, 'csv');
-            expect(exportAdvisoriesCSV).toHaveBeenCalledWith(
-                {
-                    page: 1,
-                    page_size: 20
-                },
-                'advisories'
-            );
-        });
-
-        it('Should download json file', () => {
-            wrapper.find('TableView').props().onExport(null, 'json');
-            expect(exportAdvisoriesJSON).toHaveBeenCalledWith(
-                {
-                    page: 1,
-                    page_size: 20
-                },
-                'advisories'
-            );
-        });
-    });
-
-    it('should clear notifications store on unmount', async () => {
-        let tempWrapper;
-        try {
-            await act(async () => {
-                tempWrapper = mount(
-                    <Provider store={store}>
-                        <Router><Advisories /></Router>
-                    </Provider>
-                );
-            });
-        } catch { console.log('Advisories component failed to mount'); }
-
-        act(() => {
-            tempWrapper.unmount();
-        });
-        const dispatchedActions = store.getActions();
-        expect(dispatchedActions.filter(
-            item => item.type === '@@INSIGHTS-CORE/NOTIFICATIONS/CLEAR_NOTIFICATIONS')
-        ).toHaveLength(1);
-    });
-
-    it('should fetch all the data using limit=-1', () => {
-        const onSelect = wrapper.find('TableView').props().onSelect;
-        try {
-            onSelect('all');
-        } catch {
-            console.log('Advisories select failed');
+    testBulkSelection(
+        fetchIDs,
+        '/ids/advisories',
+        'selectAdvisoryRow',
+        {
+            id: 'RHEA-2020:2743',
+            selected: 'RHEA-2020:2743'
         }
+    );
+
+    it('Should display error page when status is rejected', async () => {
+        const rejectedState = {
+            ...mockState,
+            status: { isLoading: false, code: 400, hasError: true },
+            error: {}
+        };
+        const tempStore = initStore(rejectedState);
+        const { container } = render(<ComponentWithContext renderOptions={{ store: tempStore }}>
+            <Advisories/>
+        </ComponentWithContext>);
 
-        expect(fetchIDs).toHaveBeenCalledWith(
-            '/ids/advisories',
-            { limit: -1, offset: 0, page: 1, page_size: 20 }
+        await waitFor(
+            () => expect(queryByText(container, 'This page is temporarily unavailable')).toBeVisible()
```

Comment:
Avoid using container. You can just use
```suggestion
            () => expect(screen.queryByText('This page is temporarily unavailable')).toBeVisible()
```
here